src/app/app-extras.module.ts

```diff
@@ -10,10 +10,17 @@ import {
   SkyThemeModule
 } from './public';
 
+import {
+  SkyAppStyleLoader
+} from './public/style-loader';
+
 @NgModule({
   exports: [
     SkyThemeModule,
     SkyMediaQueryModule
+  ],
+  providers: [
+    SkyAppStyleLoader
```

Comment:
The `SkyAppStyleLoader` is provided by the `SkyThemeModule`, so this doesn't need to be added here?

Reply:
Good catch.  Removed.